integrations/integration_cname.js

```diff
@@ -135,6 +136,7 @@ const commonNames = {
   ...Refiner,
   ...YandexMetrica,
   ...Qualaroo,
+  all: "All",
```

Comment:
```suggestion
  All: "All",
```

Reply:
The fix is applied in a different location.